### DISCUSSION TITLE
AdaptiveCard (ver 1.3) Action.OpenUrl not working in PnP Search Results

### DISCUSSION THREAD

**Discussion by gretchunkim** [microsoft-search/pnp-modern-search#2856] [Q&A] [answered]
Anyone else having issue  with using AdaptiveCard  property type Action.OpenUrl not working? 

![image](https://user-images.githubusercontent.com/8564035/225449132-c3b22288-0a95-428e-b46f-4f6397b73651.png)

Using PnP Search Result Version 4.8.0 
![image](https://user-images.githubusercontent.com/8564035/225449066-74d08866-0198-401e-8352-8f08f8d761af.png)

**Comment by kasperbolarsen** [comment, marked as answer]
Yes, it doesn't work in the Microsoft search Result Types in the Search and Intelligence admin center either.

**Reply by gretchunkim** [reply]
Thanks @kasperbolarsen! Found a workaround, using a href instead.

**Reply by wobba** [reply]
I believe this should work:

```json
{
            "type": "Action.OpenUrl",
            "title": "View",
            "url": "https://something"
}
 ```

so could be the selectAction part which is not working. Does that work in Viva Connections and SPFx? Should be the same card rendered.

**Reply by kasperbolarsen** [reply]
I just noted that the [Search Layout designer]((https://searchlayoutdesigner.azurewebsites.net/) ) did have an with that:

![image](https://user-images.githubusercontent.com/20593570/225583182-823c40ae-4cc9-48fc-b5c4-e2cb72e2d5a1.png)

and please, don't tell me that I can't trust the designer ;-)

**Reply by wobba** [reply]
You are right. You have to use textblock and markdown for links.

```json
{
    "type": "TextBlock",
    "text": "[my text](https://something)",
    "color": "accent",
    "size": "medium",
    "weight": "bolder"
}
 ```

**Reply by gretchunkim** [reply]
Great! More options. thanks @wobba

**Reply by kasperbolarsen** [reply]
@gretchunkim feel free to reach out on Twitter regarding MS Search questions/issues. As far as I can tell no many many has started working with Result types and the like yet.

**Reply by gretchunkim** [reply]
@kasperbolarsen Sure thing! It's hard for me leave this GitHub solution space especially when the solution is built brilliantly. If you don't mind, I'd love to expand the discussion section here so it can be shared among others.
